docs/tutorials/contracts/fts/simple-fts.md

```diff
@@ -0,0 +1,155 @@
+---
+id: simple-fts
+title: Minting FTs (Simple)
+sidebar_label: Minting FTs (Simple)
+---
+
+In this tutorial, you'll learn how to easily create your own Fungible Tokens (FT) without doing any software development by using a readily-available FT smart contract.
+
+## Overview
+
+This article will show you how to use an existing [FT smart contract](#fungible-token-contract), and you'll learn [how to mint](#minting-your-fts) fungible tokens and view them in your Wallet.
+
+## Prerequisites
+
+To complete this tutorial successfully, you'll need:
+
+- [A NEAR account](#wallet)
+- [NEAR command-line interface](/docs/develop/contracts/rust/intro#installing-the-near-cli) (`near-cli`)
+
+### Wallet
+
+To store your fungible tokens you'll need a [NEAR Wallet](https://wallet.testnet.near.org/).
+If you don't have one yet, you can create one easily by following [these instructions](/docs/develop/basics/create-account):
+
+> **Tip:** If you already have a NEAR `testnet` account, you can [skip these steps](#installing-the-near-cli).
+
+#### 1. Reserve an Account ID
+
+- Navigate to https://wallet.testnet.near.org and click on "Create Account".
+- Next, enter your desired account name.
+
+#### 2. Secure your account
+
+- Choose your account recovery method. For simplicity, in this tutorial you can select
+  "E-mail Account Recovery", although "Recovery Phrase" and [Ledger](https://www.ledger.com/)
+  are the most secure methods.
+
+#### 3. E-mail / Phone Number Account Recovery
+
+- Enter the account activation code that you received.
+
+#### 4. Success!
+
+- You just created a `testnet` account and received 200 Ⓝ! Upon recovery method confirmation
+  you should be directed to your account dashboard.
+
+Once you have your Wallet account, you can click on the [Balances Tab](https://wallet.testnet.near.org/?tab=balances) where all your Fungible Tokens will be listed:
+
+![Wallet](/docs/assets/fts/empty-wallet-ft-tab.png)
```

Comment:
We don't need to have instructions on creating a wallet here. We can simply link to this process in the pre-req section. That way we avoid duplicate information in docs and eliminate the need to maintain multiple copies.

Reply:
Fixed